src/main/scala/org/globalforestwatch/summarystats/carbonflux/CarbonFluxGridSources.scala

```diff
@@ -50,56 +51,56 @@ case class CarbonFluxGridSources(gridTile: GridTile) extends GridSources {
   val jplAGBExtent = JplAGBextent(gridTile)
   val fiaRegionsUsExtent = FiaRegionsUsExtent(gridTile)
 
-  def readWindow(window: Extent): Either[Throwable, Raster[CarbonFluxTile]] = {
+  def readWindow(windowKey: SpatialKey, windowLayout: LayoutDefinition): Either[Throwable, Raster[CarbonFluxTile]] = {
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)